components/base-rest-api/packages/services/lib/authentication-service.js

```diff
@@ -94,6 +97,11 @@ class AuthenticationService extends Service {
         { token, issuer: claims.iss },
         providerConfig,
       );
+      const tokenSwapperService = await this.service('tokenSwapperService');
+      await tokenSwapperService.swap({
+        token,
+        uid,
+      });
```

Comment:
Every time I make an API call is this `init` method called? Meaning this `swap` is call on every API interaction?